core/src/main/java/de/bwaldvogel/mongo/MongoDatabase.java

```diff
@@ -32,9 +32,9 @@
 
     MongoCollection<?> resolveCollection(String collectionName, boolean throwIfNotFound);
 
-    void drop();
+    void drop(Oplog oplog);
```

Comment:
any reason we can't pass the `OpLog` in the constructor of the Collection impl e.g. `MemoryCollection`?

Reply:
you mean the constructor of AbstractDatabase?
I think we could.